components/community/HomeCard.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { ButtonSize } from '@/types/components/buttons/ButtonPropsType';
+import { HeadingLevel, HeadingTypeStyle } from '@/types/typography/Heading';
+
+import Button from '../buttons/Button';
+import Heading from '../typography/Heading';
+
+interface HomeCardProp {
+  headline: string;
+  title: string;
+  description: string;
+  btnText: string;
+  link: string;
+  className: string;
+};
+
+/**
+ * @description This component displays Home Card.
+ * @param {HomeCardProp} props - The props for Home Card component.
+ * @param {string} props.headline - The headline for the home card.
+ * @param {string} props.title - The title for the home card.
+ * @param {string} props.description - The description for the home card.
+ * @param {string} props.btnText - The text for the button in the home card.
+ * @param {string} props.link - The link for the button in the home card.
+ * @param {string} props.className - Additional CSS classes for styling..
+ */
+export default function HomeCards({
+  headline,
+  title,
+  description,
+  btnText,
+  link,
+  className
+} : HomeCardProp) : React.ReactNode {
```

Comment:
```suggestion
} : HomeCardProp) {
```

No need to define return type for the functional components.

Reply:
Sorry for this. I have rectified this now.